### ISSUE TITLE
[FEA] Add a json_tuple kernel

### ISSUE THREAD

**Issue by revans2** [NVIDIA/spark-rapids-jni#2036]
**Is your feature request related to a problem? Please describe.**
We support `get_json_object`, which is great, and we have implemented `json_tuple` in terms of `get_json_object`. But this is problematic because we have to run `get_json_object ` multiple times. Once for each column we want to pull out. It would be much better if we just had a kernel that could do multiple columns at the same time, and it would not need to worry about the complexity of JSON PATH as it is just looking for top level columns.

**Comment by jihoonson** [comment]
Hi @revans2, is this issue duplicated with https://github.com/NVIDIA/spark-rapids/issues/11386?

**Comment by revans2** [comment]
Yes we can close this in favor of https://github.com/NVIDIA/spark-rapids/issues/11386. But I don't consider it a duplicate. This was filed before we had the multi-get_json_object, so this is really just out of date.

We could still create a custom kernel just for json_tuple, but I don't know how much of a performance improvement it would being.
